homeassistant/components/drop_connect/sensor.py

```diff
@@ -222,6 +223,7 @@ class DROPSensorEntityDescription(SensorEntityDescription):
     ],
     DEV_FILTER: [BATTERY, CURRENT_FLOW_RATE, CURRENT_SYSTEM_PRESSURE],
     DEV_LEAK_DETECTOR: [BATTERY, TEMPERATURE],
+    DEV_ALERT: [BATTERY, TEMPERATURE],
```

Comment:
This name should perhaps be something more than just sensor or even just use the device class as its name?

Reply:
Could you clarify the nature of the change you're suggesting?  I agree the name of the product ("Alert") and the monitored sensor ("Sensor") are rather ambiguous, but those names were specified by our marketing group and at this point I can't change anything that is customer-facing.  What users see in HA needs to match what they see in our mobile apps.  The generic naming stems from the fact that this product has a wide variety of potential applications.